webclient/pom.xml

```diff
@@ -217,10 +217,10 @@
         <plugins>
             <plugin>
                 <artifactId>maven-compiler-plugin</artifactId>
-                <version>2.4</version>
+                <version>3.3</version>
```

Comment:
plugin version is duplicated. Should be placed in the parent pom in pluginManagement section.